lib/environ/src/cranelift.rs

```diff
@@ -110,10 +136,21 @@ pub fn compile_module<'data, 'module>(
         context
             .compile_and_emit(isa, &mut code_buf, &mut reloc_sink, &mut trap_sink)
             .map_err(CompileError::Codegen)?;
+
+        align_code_buf(&mut code_buf);
+
+        let body_len = code_buf.len();
+        let at = get_address_transform(&context, isa);
```

Comment:
Can this be made conditional, so that we don't create these tables if `!context.debug_info()`?